core/baseline-core-v1.0-psd01.md

```diff
@@ -2946,14 +3013,20 @@ This document lists the requirements for either option and indicates the differe
 * *Support for database clustering to process high data volumes in short periods of time (sub second processing)*
 * *Security features that adhere to established industry security standards such as the US Federal Information Processing Standard [[FIPS](#fips)] or [[ISO 27001](#iso27001)]*  
 
+[[R309]](#r309) Testability: An example of a BPI consistent with the requirement is the open-source [PostgreSQL](https://www.postgresql.org/) data base, and since PostgreSQL has an extensive test suite in line with enterprise-grade data base solutions, the requirement is testable.
```

Comment:
What's the basis for claiming that PostgreSQL is consistent with the requirement?

The existence of a test suite seems entirely irrelevant to the claim of testability.

Is there any way to generalise this further than "if you use PostgreSQL you successfully meet this requirement"?

Reply:
@chaals Not only postgres, of course, but yes, there is not really a way to generalize this. This is a "mollify enterprises" requirement. One could add .. and has undergone enterprise and government certification processes".

Reply:
I'm not sure that "enterprises are happy" belongs in a spec. If the spec is good enough, enterprises will be happy to use it. So the requirements should be clear enough to make a decision on whether an implementation meets them. Nothing stops an implementation doing other things to make an enterprise happy, if that seems necessary. If it's a particular thing that lots want, make it a formal requirement, otherwise, leave it to the market - different users will have different levels of happiness with different implementation approaches.

Reply:
I added a reference to a multi-certificate (enterprise, security, regulation) MSFT Azure PostgreSQL implementation that meets the mentioned requirements @chaals that should do it.